auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/Crn.java

```diff
@@ -226,7 +227,8 @@ public String toString() {
         NAMESPACE("namespace"),
         KCLUSTER("kcluster"),
         SAML_PROVIDER("samlProvider"),
-        WORKSPACE("workspace");
+        WORKSPACE("workspace"),
+        DATABASE("database");
```

Comment:
Looks like I should add databaseserver, and implement CRN support generally for them.

Reply:
I think yes, that was the idea. I added a CrnService, there DatabaseServer stuff can also be added, if you see it fit.